CMakeLists.txt

```diff
@@ -42,6 +42,10 @@ set(TRITON_BACKEND_REPO_TAG "main" CACHE STRING "Tag for triton-inference-server
 set(TRITON_COMMON_REPO_TAG "main" CACHE STRING "Tag for triton-inference-server/common repo")
 set(TRITON_CORE_REPO_TAG "main" CACHE STRING "Tag for triton-inference-server/core repo")
 
+if(NOT CMAKE_BUILD_TYPE)
```

Comment:
Why is this required?

Reply:
Adding this just because I'm seeing we set CMAKE_BUILD_TYPE for other backends but not for Python backend. I can remove it if you think this is not needed.